test/e2e/specs/search/search__jetpack-instant.ts

```diff
@@ -24,151 +24,151 @@ declare const browser: Browser;
+				( response ) =>
+					response
+						.url()
+						.includes( 'jetpack-search/build/instant-search/jp-search.chunk-main-payload.js' ),
+				{ timeout: 30 * 1000 }
+			);
+
+			await page.goto( postWithSearchBlockUrl, {
 				timeout: 20 * 1000,
 				waitUntil: 'domcontentloaded',
 			} );
-		}
 
-		await waitForSearchJsPromise;
-
-		searchModalComponent = new JetpackInstantSearchModalComponent( page );
-	} );
-
-	it( 'Enter search term and launch search modal', async function () {
-		// Sometimes the parent block on a homepage has a very high-up aria-hidden.
-		// TODO: figure out what is adding that, and remove the "includeHidden" here.
-		const inputLocator = page
-			.getByRole( 'search', { includeHidden: true } )
-			.getByRole( 'searchbox', { name: 'Search', includeHidden: true } )
-			.first();
-		const buttonLocator = page
-			.getByRole( 'search', { includeHidden: true } )
-			.getByRole( 'button', { name: 'Search', includeHidden: true } )
-			.first();
-
-		// Adding a slightly longer timeout here because we can't fully wait for the "load" event above due to
-		// a collision with WordAds. This helps share some of the initial load wait with the first interaction.
-		await inputLocator.fill( searchString, { timeout: 20 * 1000 } );
-		await Promise.all( [
-			searchModalComponent.expectAndWaitForSearch( searchString ),
-			buttonLocator.click(),
-		] );
-	} );
-
-	it( 'The search term pulls into the modal', async function () {
-		// See: https://github.com/Automattic/jetpack/issues/32753
-		// There's a rare race condition where the spaces get URL encoded as "+" and that pulls into the modal.
-		// We don't wait to fail on that, so accounting for it specifically here.
-		const termInModal = ( await searchModalComponent.getSearchTerm() ).replace( /\+/g, ' ' );
-		expect( termInModal ).toEqual( searchString );
-	} );
-
-	skipItIf( envVariables.TEST_ON_ATOMIC )( 'There are search results', async function () {
-		expect( await searchModalComponent.getNumberOfResults() ).toBeGreaterThanOrEqual( 1 );
-	} );
-
-	skipItIf( envVariables.TEST_ON_ATOMIC )(
-		'The search term is present in the results as a highlighted match',
-		async function () {
-			await searchModalComponent.validateHighlightedMatch( searchString );
-		}
-	);
+			await waitForSearchJsPromise;
+
+			searchModalComponent = new JetpackInstantSearchModalComponent( page );
+		} );
 
-	it( 'Clear the search term', async function () {
-		await Promise.all( [
-			searchModalComponent.expectAndWaitForSearch( '' ),
-			searchModalComponent.clearSearchTerm(),
-		] );
+		it( 'Enter search term and launch search modal', async function () {
+			// Sometimes the parent block on a homepage has a very high-up aria-hidden.
+			// TODO: figure out what is adding that, and remove the "includeHidden" here.
+			const inputLocator = page
+				.getByRole( 'search', { includeHidden: true } )
+				.getByRole( 'searchbox', { name: 'Search', includeHidden: true } )
```

Comment:
**comment** - when you say "very high-up aria-hidden", does that mean the published and rendered Search block has an `aria-hidden` attribute?

When this happens, does the Search block render in the published post?

Reply:
On some of the sites in the past, there was this weird case (maybe a theme bug) where like the `main` equivalent had `aria-hidden` on it. But, I couldn't get it to reproduce on other sites. This didn't affect the actual visibility, just the a11y visibility.

I don't think this is still happening now (resetting the theme may have fixed it), so we can go back and take this out! I didn't actually add this in this PR too, this was from a while back, this just got thrown off by me adding the `skipDescribeIf` 👍